scripts/nb-tester/test-notebook.py

```diff
@@ -32,7 +32,7 @@
 ]
 NOTEBOOKS_THAT_SUBMIT_JOBS = [
     "docs/start/hello-world.ipynb",
-    "docs/analyze/saving-and-retrieving.ipynb",
```

Comment:
Fwiw, this seems like it always belonged in "execute on hardware", not postprocess/analyze